services/docker-compose.yml

```diff
@@ -236,6 +238,41 @@ services:
       - default
       - interactive_services_subnet
 
+  webserver-garbage-collector:
+    image: ${DOCKER_REGISTRY:-itisfoundation}/webserver:${DOCKER_IMAGE_TAG:-latest}
+    init: true
+    hostname: "{{.Node.Hostname}}-{{.Service.Name}}-{{.Task.Slot}}"
+    environment:
+      <<: *webserver-environment
+      WEBSERVER_GARBAGE_COLLECTOR: '{"GARBAGE_COLLECTOR_INTERVAL_S": 30}'
+    env_file:
+      - ../.env
+      - ../.env-webserver-garbage-collector
+
+    healthcheck:
+      test:
+        [
+          "CMD",
+          "python3",
+          "/home/scu/services/web/server/docker/healthcheck.py",
+          "http://localhost:8080/v0/"
+        ]
+      interval: 30s
+      timeout: 120s
+      retries: 3
+      start_period: 30s
+
+    deploy:
+      placement:
+        constraints:
+          - node.platform.os == linux
+      labels:
+        - io.simcore.zone=${TRAEFIK_SIMCORE_ZONE}
```

Comment:
yes, traffik should not
Nonetheless, notice that the rest plugin is active because this service **needs** one of the routes ``/v0/`` for the swarm healthcheck.

@mrnicegyu11 @Surfict i guess you also need to make sure in ops that this service has correct deploy rules?